src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__gradebook_gpa.sql

```diff
@@ -0,0 +1,664 @@
+        where
+            enr.academic_year >= {{ var("current_academic_year") - 1 }}
+            and enr.rn_year = 1
+            and not enr.is_out_of_district
+            and enr.grade_level != 99
+    ),
+
+    course_enrollments as (
+        select
+            m._dbt_source_relation,
+            m.cc_studentid as studentid,
+            m.cc_yearid as yearid,
+            m.cc_course_number as course_number,
+            m.cc_sectionid as sectionid,
+            m.cc_dateenrolled as date_enrolled,
+            m.sections_dcid,
+            m.sections_section_number as section_number,
+            m.sections_external_expression as external_expression,
+            m.sections_termid as termid,
+            m.courses_credittype as credit_type,
+            m.courses_course_name as course_name,
+            m.courses_excludefromgpa as exclude_from_gpa,
+            m.teachernumber as teacher_number,
+            m.teacher_lastfirst,
+
+            f.tutoring_nj,
+            f.nj_student_tier,
+        from {{ ref("base_powerschool__course_enrollments") }} as m
+        left join
+            {{ ref("int_reporting__student_filters") }} as f
+            on m.cc_studentid = f.studentid
+            and m.cc_academic_year = f.academic_year
+            and m.courses_credittype = f.powerschool_credittype
+            and {{ union_dataset_join_clause(left_alias="m", right_alias="f") }}
+        where
+            m.rn_course_number_year = 1
+            and not m.is_dropped_section
+            and m.cc_course_number not in (
+                'LOG100',
+                'LOG1010',
+                'LOG11',
+                'LOG12',
+                'LOG20',
+                'LOG22999XL',
+                'LOG300',
+                'LOG9',
+                'SEM22106G1',
+                'SEM22106S1'
+            )
+    ),
+
+    y1_historical as (
+        select
+            tr._dbt_source_relation,
+            tr.studentid,
+            tr.academic_year,
+            tr.yearid,
+            tr.schoolid,
+            tr.schoolname,
+            tr.grade_level,
+            tr.course_name,
+            tr.sectionid,
+            tr.teacher_name,
+            tr.storecode,
+            tr.termid,
+            tr.percent,
+            tr.grade,
+            tr.earnedcrhrs,
+            tr.potentialcrhrs,
+            tr.gpa_points,
+            tr.excludefromgpa,
+            tr.is_transfer_grade,
+
+            if(tr.is_transfer_grade, 'Transfer', tr.credit_type) as credit_type,
+            if(
+                tr.is_transfer_grade,
+                concat(
+                    'T',
+                    upper(regexp_extract(tr._dbt_source_relation, r'(kipp\w+)_')),
+                    tr.dcid
+                ),
+                tr.course_number
+            ) as course_number,
+
+            if(co.student_number is not null, true, false) as is_enrollment_matched,
+        from {{ ref("stg_powerschool__storedgrades") }} as tr

```

Comment:
was originally filtered to current year. dropped to include Y1 historical (including transfer grades) but not sure if that was necessary